oh_staff_ui/models.py

```diff
@@ -287,3 +287,61 @@ class ItemCopyrightUsage(models.Model):
     type = models.ForeignKey(
         CopyrightType, on_delete=models.PROTECT, blank=False, null=False
     )
+
+
+class RelationType(models.Model):
+    type = models.CharField(max_length=256, blank=False, null=False)
+
+    def __str__(self):
+        return self.type
+
+
+class Relation(models.Model):
+    value = models.CharField(max_length=1024, blank=False, null=False)
+
+    def __str__(self):
+        return self.value
+
+
+class ItemRelationUsage(models.Model):
+    item = models.ForeignKey(
+        ProjectItem, on_delete=models.PROTECT, blank=False, null=False
+    )
+    relation = models.ForeignKey(
+        Relation, on_delete=models.PROTECT, blank=False, null=False
+    )
+    type = models.ForeignKey(
+        RelationType, on_delete=models.PROTECT, blank=False, null=False
+    )
+
+
+class Format(models.Model):
```

Comment:
Same general question as for `Coverage` - if this is uncontrolled, there seems to be no real benefit in a `...Usage` table, so this could be an optional single attribute on `ProjectItem` or a standard `Format` model with `ProjectItem` added as an FK.

After discussion, since this is repeatable uncontrolled free text, we don't need the Usage table, just add item as FK to `Format`.

Reply:
Format is uncontrolled but repeatable, so will add item FK to `Format`